cloud/services/virtualnetworks/virtualnetworks.go

```diff
@@ -89,7 +89,6 @@ func (s *Service) Reconcile(ctx context.Context, spec interface{}) error {
 			s.Scope.V(2).Info("Working on custom vnet", "vnet-id", vnet.ID)
 		}
 		// vnet already exists, cannot update since it's immutable
-		// TODO: ensure tags & other managed vnet attributes
```

Comment:
**resolved**: this is not needed: if the vnet is detected as "managed" then it already has the "owner" tags. If not, then it is probably unmanaged and we shouldn't apply those tags. I don't think "other managed vnet attributes" applies here, I believe this is a leftover copy paste from capa.